compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -713,7 +717,7 @@ OMR::SymbolReferenceTable::findOrCreateCodeGenInlinedHelper(CommonNonhelperSymbo
    {
    if (!element(index))
       {
-      TR::MethodSymbol * sym = TR::MethodSymbol::create(trHeapMemory(),TR_Helper);
+      TR::MethodSymbol * sym = TR::MethodSymbol::create(trHeapMemory(),TR_None);
```

Comment:
This change should not cause behavior change since CodeGenInlinedHelper are not, and should not be, handled by any linkage; instead, CodeGen recognizes them and generates inlined assembly. Therefore, they do not use any of existing linkage convention, and hence, I believe, it is a perfect example that one helper with no linkage, noted as TR_None.